apps/web/app/api/v1/client/[environmentId]/storage/local/route.ts

```diff
@@ -108,9 +109,7 @@ export const POST = async (req: NextRequest, context: Context): Promise<Response
   }
 
   try {
-    const plan = ["active", "canceled"].includes(organization.billing.features.linkSurvey.status)
-      ? "pro"
-      : "free";
+    const plan = (await getBiggerUploadFileSizePermission(organization)) ? "pro" : "free";
```

Comment:
here the plan is `pro` and `free` which is different from what we have in the rest of the app. Can we use the real plan here or just the boolean value for the bigger file size permission to avoid confusion?